telemetry/definitions/commonDefinitions.json

```diff
@@ -1,5 +1,11 @@
 {
     "types": [
+        {
+            "name": "toolkitTelemetryStatusType",
+            "type": "string",
+            "description": "The type to determine the users status of having telemetry event submitted or not.",
+            "allowedValues": ["OFF", "ON"]
```

Comment:
is a new type needed for this or could it use this: https://github.com/aws/aws-toolkit-common/blob/ffab9084e1d0056aed79eb65ac4dffa506eb9f7d/telemetry/definitions/commonDefinitions.json#L257-L260

Reply:
If we do carry this out, I would like to re-use types. I will delete this type from the PR.